src/Sylius/Bundle/ReviewBundle/Resources/config/services.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" ?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <parameters>
+        <parameter key="sylius.event_subscriber.review.load_metadata.class">Sylius\Bundle\ReviewBundle\EventListener\LoadMetadataSubscriber</parameter>
+        <parameter key="sylius.review.validator.unique_reviewer_email.class">Sylius\Bundle\ReviewBundle\Validator\Constraints\UniqueReviewerEmailValidator</parameter>
+        <parameter key="sylius.listener.review_create.class">Sylius\Bundle\ReviewBundle\EventListener\ReviewCreateListener</parameter>
+        <parameter key="sylius.listener.review_delete.class">Sylius\Bundle\ReviewBundle\EventListener\ReviewDeleteListener</parameter>
+        <parameter key="sylius.review.calculator.average_rating.class">Sylius\Component\Review\Calculator\AverageRatingCalculator</parameter>
+        <parameter key="sylius.review.updater.average_rating.class">Sylius\Bundle\ReviewBundle\Updater\AverageRatingUpdater</parameter>
+        <parameter key="sylius.review.remover.reviewer_reviews.class">Sylius\Bundle\ReviewBundle\Remover\ReviewerReviewsRemover</parameter>
+    </parameters>
+
+    <services>
+        <!--Subscribers-->
+        <service id="sylius.event_subscriber.review.load_metadata" class="%sylius.event_subscriber.review.load_metadata.class%">
+            <argument>%sylius.review.subjects%</argument>
+            <tag name="doctrine.event_subscriber" />
+        </service>
+
+        <!--Validators-->
+        <service id="sylius.review.validator.unique_reviewer_email" class="%sylius.review.validator.unique_reviewer_email.class%">
+            <argument type="service" id="sylius.repository.user" />
```

Comment:
Now, ``CustomerGuestType`` is added in ``ReviewType``, but I guess that it is quite wrong approach and it should be added in CoreBundle to ``ProductReviewType``. It would be fixed problem with dependency between Review and User. @michalmarcinkowski @pjedrzejewski WDYT?